src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AnnotationsCacheWarmer.php

```diff
@@ -59,7 +59,7 @@ protected function doWarmUp(string $cacheDir, ArrayAdapter $arrayAdapter)
             // doctrine/annotations:1.13 and above
             $reader = new PsrCachedReader($this->annotationReader, $arrayAdapter, $this->debug);
         } else {
-            $reader = new CachedReader($this->annotationReader, new DoctrineProvider($arrayAdapter), $this->debug);
+            $reader = new CachedReader($this->annotationReader, DoctrineProvider::wrap($arrayAdapter), $this->debug);
```

Comment:
This line can be removed, right? It is dead code if we also update to `doctrine/annotations:1.13`.

Reply:
As soon as we have a stable release of doctrine/annotations 1.13, yes.

Reply:
doctrine/annotations 1.13 is not released yet and is not sure to be released before the next 5.x release.

Reply:
oops sorry. I read the wrong version on packagist. I though it was released already.

Reply:
No worries. I also misread and it looks like this is scheduled for 5.4 anyways, so we can definitely make that change.